src/main/java/io/github/externschool/planner/service/RoleServiceImpl.java

```diff
@@ -12,7 +13,7 @@ public RoleServiceImpl(RoleRepository roleRepository) {
     }
 
     @Override
-    public io.github.externschool.planner.entity.Role getRoleByName(String name) {
+    public io.github.externschool.planner.entity.Role getRoleByName(String name) throws RoleNotFoundException {
         return roleRepository.findByName(name);
     }
```

Comment:
1. Thanks for this inline export discovered, move it to the top please, so we get  `public Role getRoleByName(String name) throws RoleNotFoundException`  method.
2. As regards to the exception usage. I doubt it will ever launch itself, so we have to do it like Oracle recommends here, for example: https://docs.oracle.com/javase/tutorial/essential/exceptions/throwing.html